esprit/tasks.py

```diff
@@ -96,6 +96,57 @@ def dump(conn, type, q=None, page_size=1000, limit=None, method="POST", out=None
             record = transform(record)
         out.write(json.dumps(record))
 
+
+def create_alias(conn, alias):
+    actions = raw.to_alias_actions(add=[{"alias": alias, "index": conn.index}])
+    print raw.post_alias(conn, actions).json()
```

Comment:
Debug statement?

If esprit has its own logging set up (via import `logging` and setting up a logging channel for esprit) then maybe use that. Otherwise it would be nice to have this prefixed by some explanation. For example, insert a line just above this line that says `print "Posting the following to the Elasticsearch alias endpoint"`.